examples/menubar/menubar-navigation.html

```diff
@@ -200,8 +200,12 @@ <h2 id="ex_label">Example</h2>
     <h2>Accessibility Features</h2>
     <ol>
             <li>To help communicate that the arrow keys are available for directional navigation within the menubar and its submenus, a border is added to the menubar container when focus is within the menubar.</li>
-      <li>To support operating system high contrast settings, focus is highlighted by adding and removing a border around the menu item with focus.</li>
-      <li>The down arrow and checked icons are made compatible with operating system high contrast settings and hidden from screen readers by using the CSS <code>content</code> property to render images.</li>
+      <li>To support operating system high contrast settings:
+        <ul>
+          <li>Focus is highlighted by adding and removing a border around the menu item with focus.</li>
+          <li>The arrow icons used to indicate the current state of pull expandable menus use inline SVG images.</li>
```

Comment:
Probably should also mention currentColor? It's an important part of the support.
(also, probably just say "expandable" and not "pull expandable")

```suggestion
          <li>The arrow icons used to indicate the current state of expandable menus use inline SVG images with CSS currentColor.</li>
```